src/js/components/LoginForm.js

```diff
@@ -81,6 +82,13 @@ var LoginForm = React.createClass({
         </div>
       );
     }
+    
+    var usernameType = 'email';
```

Comment:
Thanks for working on this.

The React way to do this is to add a getDefaultProps() function and have it return: {usernameType: 'email'}
That way, you can just put this.props.usernameType as the value for the input type attribute.